OWML.Launcher/App.cs

```diff
@@ -162,6 +136,15 @@ private void PatchGame(IList<IModData> mods)
         private void StartGame(string[] args)
         {
             _writer.WriteLine("Starting game...");
+
+            if (args.Contains("-consolePort"))
+            {
+                var index = Array.FindIndex(args, x => x == "-consolePort");
+                var list = new List<string>(args);
+                list.RemoveRange(index, 2);
+                args = list.ToArray();
```

Comment:
args.Take(count). 
Might need .ToList() first

Reply:
wdym? he removes index and index+1 from that list

Reply:
To reduce three lines to one

Reply:
Yeah, my suggestion wouldn't work. I still don't like that it takes three lines to remove the two items (actually I don't like that we're removing them at all, it means we're using the params in completely different ways which is hacky) - I'll give this some more thought... 😋

Reply:
I guess the weird part is that we're passing args from OWML into OW. That can be adressed in another PR.